pkg/haconfig/interfaces.go

```diff
@@ -0,0 +1,17 @@
+package haconfig
```

Comment:
Yeah, I didn't have a strong preference/suggestion for the name. 
An ipfailover by any other name s(m)ells just as sweet.